controllers/student.go

```diff
@@ -44,6 +44,18 @@ type StudentDashboard struct {
 	ProjectsWorked []ProjectDashboard `json:"projects_worked"`
 }
 
+// RegisterStudent godoc
+//
+//	@Summary		Register a student
+//	@Description	Register a new student with the provided details.
+//	@Accept			json
+//	@Produce		plain
+//	@Param			request	body		RegisterStudentReqFields	true	"Fields required for student registeration"
```

Comment:
Does the RegisterStudentReqFields struct need to be annotated?

Reply:
Ig not. Not in this case at least.

Reply:
that is what creates [this](https://kossiitkgp.org/KWoC-Backend/#/paths/~1student~1form~1/post)